diff --git a/services/inputflinger/reader/EventHub.cpp b/services/inputflinger/reader/EventHub.cpp
index 5085ad6..00d41f8 100644
--- a/services/inputflinger/reader/EventHub.cpp
+++ b/services/inputflinger/reader/EventHub.cpp
@@ -2341,6 +2341,13 @@
     // Load the configuration file for the device.
     device->loadConfigurationLocked();
 
+    // Disable device if device config property set
+    if (device->configuration &&
+        device->configuration->getBool("device.disabled")) {
+        device->disable();
+        ALOGV("Disabling device with id %d\n", device->id);
+    }
+
     // Figure out the kinds of events the device reports.
     device->readDeviceBitMask(EVIOCGBIT(EV_KEY, 0), device->keyBitmask);
     device->readDeviceBitMask(EVIOCGBIT(EV_ABS, 0), device->absBitmask);
